### PR TITLE
fixes #87 - adds assertions for lics & updates tests

### LINKED ISSUE #87: Add variable bound assertions for lics
Add variable bound assertions for lics, ensuring complete LIC logic

### PR DESCRIPTION
**Pull request by sasoder**

### CODE REVIEW

File: cmv.py [hunk 1]
```diff
@@ -241,6 +258,10 @@ def lic_9(POINTS: list, NUMPOINTS: int, C_PTS: int, D_PTS: int, EPSILON: float):
 
 # LIC 10
 def lic_10(E_PTS: int, F_PTS: int, AREA1: float, POINTS: list, NUMPOINTS: int):
+    assert E_PTS >= 1, "E_PTS must be at least 1"
+    assert F_PTS >= 1, "F_PTS must be at least 1"
+    assert E_PTS + F_PTS <= NUMPOINTS - 3, "E_PTS + F_PTS must be at most NUMPOINTS-3"
+    assert AREA1 >= 0, "AREA1 must be non-negative"
     if E_PTS < 1 or F_PTS < 1:
```
Comment (peschwartz) [inline cmv.py:265]: should you delete these checks? The assertions already check the values so you don't need to do it again

File: test/testcmv.py
Comment (peschwartz) [inline test/testcmv.py:1]: these tests look good, thanks! 

do you think we need tests for empty lists of points? Or NUMPOINTS being 0/negative (if there is not already a check for NUMPOINTS)

Comment (sasoder) [reply test/testcmv.py:1]: I think we have done more than enough
